apps/video/test/marionette/info_view_test.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+var Video = require('./lib/video'),
+    assert = require('assert'),
+    VideoOptionsView = require('./lib/video_options_view.js'),
+    VideoInfoView = require('./lib/video_info_view.js');
+
+marionette('video info view', function() {
+  var client = marionette.client({
+    profile: {
+      prefs: {
+        'device.storage.enabled': true,
+        'device.storage.testing': true,
+        'device.storage.prompt.testing': true
+      }
+    },
+    desiredCapabilities: { raisesAccessibilityExceptions: false }
+  });
+  var app, selectors, optionsView, infoView;
+
+  setup(function() {
+    // Remove all files in temp device storage.
+    client.fileManager.removeAllFiles();
+    
+    // Add file into the videos directory
+    client.fileManager.add({
+      type: 'videos',
+      filePath: 'test_media/Movies/elephants-dream.webm'
+    });
+
+    selectors = Video.Selector;
+    app = new Video(client);
+    optionsView = new VideoOptionsView(client);
+    infoView = new VideoInfoView(client);
+  });
+
+  test('open video and check video info', function() {
+    // tap thumbnail to open video
+    app.launch();
+    client.helper.waitForElement(selectors.thumbnail);
+    app.waitForThumbnails(1);
+    app.playVideoFromGalleryView();
+    app.waitForVideoToStartPlaying();
+    
+    // tap screen to show menu, then tap option
+    app.tapOption();
+    assert.equal(optionsView.optionsViewDisplayed, true);
+    
+    // tap more info from options menu
+    optionsView.tapMoreInfo();
+    assert.equal(infoView.infoViewDisplayed, true);
```

Comment:
It seems there can be a timing issue here where 'infoView.infoViewDisplayed' is tested before the info view is actually displayed. Do you agree? I've also added a comment below in the 'video_options_view.js' section regarding this issue.